Dockerfile

```diff
@@ -1,25 +1,74 @@
-FROM node:22.16 as build
-ENV NODE_VERSION=22.14.0
+ARG CONTEXT=prod
+FROM node:22.16-alpine as base
 
+## Setup
+ARG CONTEXT
+ENV NODE_VERSION=22.16.0
 WORKDIR /app
+ENV ${CONTEXT}=1
 
+## Installs
 COPY client/package.json .
 COPY client/package-lock.json .
 RUN npm ci
 
-COPY client /app/
+COPY client ./
+
+## External Information
+LABEL org.opencontainers.image.title="OpenSlides Client"
+LABEL org.opencontainers.image.description="Web client for OpenSlides which serves as the users main interaction point while using the OpenSlides system. "
+LABEL org.opencontainers.image.licenses="MIT"
+LABEL org.opencontainers.image.source="https://github.com/OpenSlides/openslides-client"
+
+## Command
+COPY ./dev/command.sh ./
+RUN chmod +x command.sh
+CMD ["./command.sh"]
+
+
+# Development Image
+FROM base as dev
+
+RUN apk add --no-cache git
+
+
+# Testing Image
+
+FROM dev as tests
+
+# Playwright Image
+
+FROM mcr.microsoft.com/playwright:v1.52.0-jammy as playwright
+
+WORKDIR /
+
+COPY ./package.json .
+COPY ./package-lock.json .
+ENV CI=1
+RUN npm ci
```

Comment:
I don't think that this should be included in the main `Dockerfile`. The integration tests are not really part of the client.